src/components/CreateWallet.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <main class="main pb-4">
+    <div class="container-fluid top-bar"></div>
+    <div class="container mt-5">
+      <section class="top-column column">
+        <h1 class="welcome-user">Créer mon portefeuille</h1>
+      </section>
+      <!-- conteneur des colonnes de gauche et droite où se situent les cards -->
+      <!--<div class="column is-flex is-justify-content-space-between pl-0 pr-0">
+        <LeftColAdmin />
+        <RightColAdmin />
+      </div> -->
+    </div>
+  </main>
+</template>
+
+<script lang="ts">
+
+import {defineComponent} from "vue"
+import {useStore} from "vuex"
+
+export default defineComponent({
+    name:"Admin",
+ 
+    setup(): any {
+      const store : any = useStore()
+      return {
+        store: store
+      }
+    },
+    data():any {
+      const store : any = useStore()
+      return {
+        userProfile: store.getters.getUserProfile()
+      }
+    },
+
+  mounted() {
```

Comment:
What is the rationale for this part ? it seems copy/pasted, and actually there's a bug inside (I fixed it in a commit not yet sent, so no problem.) My question is why do you look for autologin user in this screen at mount ? I'm not saying it is a bad idea, just thinking out loud. There are 2 principles we can think about that are maybe useful : 1 - lazy loading : asking for auth and login only when we need it, 2 - Don't repeat yourself ... this code is duplicated, so even if we want this behavior, it should probably be in some master component (`Nav.vue` ?), and only once. What are you thoughts on that ?

Reply:
Yes, actually I think that we already are logged in, no need to re-login. Just pull the userprofile in a computer property from the store, which already contains it.